Source/Scene/Points3DTileContent.js

```diff
@@ -148,19 +155,42 @@ define([
         }
     });
 
+    function createFeatures(content) {
```

Comment:
Can you also add a comment here about this so readers don't assume a one-to-one relationship between points-and-features.